presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -366,36 +366,6 @@ protected Scope visitCreateView(CreateView node, Scope scope)
         return createScope(node, scope, queryScope.getRelationType());
     }
 
-    @Override
-    protected Scope visitDataDefinitionStatement(DataDefinitionStatement node, Scope scope)
-    {
-        return createScope(node, scope, emptyList());
-    }
-
-    @Override
-    protected Scope visitSetSession(SetSession node, Scope scope)
```

Comment:
You will still need to override these visit methods so that a scope gets created for these nodes.  (The DefaultTraversalVisitor processes their fields otherwise)